qapractice3/src/test/java/dataDriven/Student.java

```diff
@@ -0,0 +1,98 @@
+package dataDriven;
+
+public class Student {
+	
+	private String f_name;
+	private String l_name;
+	private String email;
+	private String gender;
+	private String mobile_number;
+	private String dob;
+	private String subjects;
+	private String hobbies;
+	private String address;
+	private String state;
+	private String city;
+	
+	public Student(String f_name, String l_name, String email, String gender, String mobile_number, String dob, String subjects, String hobbies, String address, String state, String city)
```

Comment:
I wouldn't bother with putting a constructor in a POJO. It can become a maintenance problem, plus it makes all fields in the class required.